### DISCUSSION TITLE
How to get Offroad maps for Enduro

### DISCUSSION THREAD

**Discussion by Advtrack** [osmandapp/OsmAnd#18242] [Maps Related]
Hello All,

I'm an offroad motorbiker using Osmand and I'm looking to get maps showing tracks and offroads authorized for traffic in France?
Also, It seems possible in Osmand do have tracks showing bigger and better than normal roads - any idea how I set this up?

Many thanks for help...

Christophe

**Comment by cricri-du-lauragais** [comment]
try this
https://github.com/OsmAnd-Rendering/Motorcycle#readme
